tensorboard/components/tensor_widget/string-utils-test.ts

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {expect} from 'chai';
+
+import {
+  ELLIPSES,
+  formatTensorName,
+  TENSOR_NAME_LENGTH_CUTOFF,
+} from './string-utils';
+
+function stringRepeat(str: string, times: number) {
+  let output = '';
+  for (let i = 0; i < times; ++i) {
+    output += str;
+  }
+  return output;
+}
+
+describe('formatTensorName', () => {
```

Comment:
Include test for including suffix behavior.
abcdefghijklmnopqrstuvwxyz -> abcdef...xyz

Reply:
Done.